lib/setup/TokenRules.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const AbiBinProvider = require('../AbiBinProvider');
+const Deployer = require('../../utils/deployContract');
+
+const ContractName = 'TokenRules';
+
+/**
+ * Helper method which performs deployment of Token Rules contract.
+ */
+class TokenRules {
+  constructor(auxiliaryWeb3, address) {
+    const oThis = this;
+    oThis.auxiliaryWeb3 = auxiliaryWeb3;
+    oThis.address = address;
+    oThis.abiBinProvider = new AbiBinProvider();
+  }
+
+  /**
+   * Performs deployment of TokenRules contract.
+   *
+   * @param _organization Organization which holds all the keys needed to administer the economy.
+   * @param _token EIP20Token address.
+   * @param txOptions Tx options.
+   */
+  async deploy(_organization, _token, txOptions) {
+    const oThis = this;
+
+    let tx = oThis._deployRawTx(_organization, _token, txOptions);
+
+    let txReceipt;
+    let params = {};
+    params.contractName = "TokenRules";
+    params.tx = tx;
+    params.txOptions = txOptions;
+    params.web3 = oThis.auxiliaryWeb3;
+
+    txReceipt = await new Deployer(params).deploy();
+
+    return txReceipt;
+  }
+
+  /**
+   * Performs deployment of TokenRules contract.
+   *
+   * @param _organization Organization which holds all the keys needed to administer the economy.
+   * @param _token EIP20Token address.
```

Comment:
_token EIP20 token contract address deployed for an economy.

For these contract variables, documentation can be copied from contracts